src/tests/gov/nasa/jpf/test/java/concurrent/ConcurrentSkipListMapTest.java

```diff
@@ -43,15 +43,15 @@ public void testFunctionality() throws Exception {
         map.put(3, 1);
       });
       t1.start();
-      t2.start();
+      // t2.start();
```

Comment:
This comments out the code that starts the second thread, making the test sequential. Please uncomment this line (and `t2.join();` below as well).